README.md

```diff
@@ -64,8 +64,7 @@ Tested on CentOS 6 and 7.
 
 You can generate a DEB package with cpack:
 ```
-sudo apt-get -y install build-essential cmake libipset-dev libmosquitto-dev
-
+sudo apt-get -y install build-essential cmake libipset-dev libmosquitto-dev pkg-config
```

Comment:
This change is unrelated, and should go in another commit. Ideally also add it for the CentOS/RedHat instructions.